osi_detectedobject.proto

```diff
@@ -161,4 +223,99 @@ message DetectedObject
         //
         optional double prob_animal = 9;
     }
+
+    // Definition of measurement states.
+    //
+    enum MeasurementState
+    {
+        // Measurement state is unknown (must not be used in ground truth).
+        //
+        MEASUREMENT_STATE_UNKNOWN = 0;
+        
+        // Measurement state is unspecified (but known, i.e. value is not part of this enum list).
+        //
+        MEASUREMENT_STATE_OTHER = 1;
+        
+        // Traffic sign has been measured by the sensor in the current timestep.
+        //
+        MEASUREMENT_STATE_MEASURED = 2;
+        
+        // Traffic sign has not been measured by the sensor in the current timestep. Values provided by tracking only.
+        //
+        MEASUREMENT_STATE_PREDICTED = 3;
+    }
+
+    //
+    // \brief The header attributes of each detected object.
+    //
+    message DetectedObjectHeader
```

Comment:
Maybe also move the DetectedObjectHeader definition out of DetectedObject, to make reuse easier?